util/dvsim/FlowCfg.py

```diff
@@ -51,15 +51,15 @@ def __init__(self, flow_cfg_file, proj_root, args):
         # are overridden with the override values.
         self.overrides = []
 
-        # List of cfgs if the parsed cfg is a master cfg list
+        # List of cfgs if the parsed cfg is a primary cfg list
         self.cfgs = []
 
-        # Add a notion of "master" cfg - this is indicated using
+        # Add a notion of "primary" cfg - this is indicated using
         # a special key 'use_cfgs' within the hjson cfg.
-        self.is_master_cfg = False
+        self.is_primary_cfg = False
```

Comment:
```suggestion
        self.is_cfg_group = False
```
@msfschaffner, @rswarbrick, @cindychip any objections to this?

Reply:
Maybe something more like "cfg list" would be better? I feel like "cfg_group" refers to a group of configurations, rather than a list of their names. But I don't have strong objections: either is fine with me.

Reply:
I'm fine either way, will let you guys decide. But @sriyerg please lmk how to word some of the other ones, such as:

`# For a non-primary cfg, it is ...`